static/app/views/issueDetails/actions/index.spec.tsx

```diff
@@ -177,6 +177,34 @@ describe('GroupActions', function () {
     });
   });
 
+  it('delete for issue platform is enabled', async () => {
+    const org = OrganizationFixture({
+      ...organization,
+      access: [...organization.access, 'event:admin'],
+      features: [...organization.features, 'issue-platform-deletions-ui'],
```

Comment:
here you use `issue-platform-deletions-ui` while in the code you use `issue-platform-deletion-ui`